prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div><a href="../prep-html-links\contact-us\contact-us.html">Click here to go to the contact page</a></div>
```

Comment:
You can remove the `../prep-html-links` part of this `href`, since your `index.html` file is in the same directory as your `contact-us/` directory.

Reply:
Gotcha! Now that i made the changes in vs code how do  upload the new code into git hub?

Reply:
Hi Bo! You'll want to repeat the add, commit, and push steps. The bot should have sent you some instructions.